### PR TITLE
Next user planning

### PR DESCRIPTION
**Pull request by MuhammadAshouri**
These changes enable planning the next data_limit and expire_date for each user with 'limited' or 'expired' statuses

### CODE REVIEW

File: app/jobs/review_users.py [hunk 1]
```diff
@@ -51,6 +57,18 @@ def review():
 
             limited = user.data_limit and user.used_traffic >= user.data_limit
             expired = user.expire and user.expire <= now_ts
+            
+            if (limited or expired):
```
Comment (M03ED) [inline app/jobs/review_users.py:61]: better to use 
```python
if (limited or expired) and user.next_user is not None:
```

File: app/telegram/handlers/report.py [hunk 2]
```diff
@@ -154,6 +154,16 @@ def report_user_usage_reset(username: str, by: str, admin: Admin = None):
...
+def report_user_data_reset_by_next(username: str, admin: Admin = None):
+    text = """  
+🔁 <b>#AutoReset</b>
+➖➖➖➖➖➖➖➖➖
+<b>Username</b> : <code>{username}</code>
+➖➖➖➖➖➖➖➖➖
+    """.format(username=escape_html(username)
+    )
+    return report(chat_id=admin.telegram_id if admin and admin.telegram_id else None, text=text)
+
```
Comment (M03ED) [inline app/telegram/handlers/report.py:166]: add more detail about user, expire and data limit, add_remaining_traffic
also add details about next user to create user and modify user report

File: app/utils/report.py [hunk 3]
```diff
@@ -141,6 +141,25 @@ def user_data_usage_reset(user: UserResponse, by: Admin, user_admin: Admin = Non
...
+        try:
+            telegram.report_user_data_reset_by_next(
+                username=user.username,
+                admin=user_admin
+            )
+        except Exception:
+            pass
+        notify(UserDataResetByNext(username=user.username, action=Notification.Type.data_reset_by_next, user=user))
+        try:
+            discord.report_user_data_reset_by_next(
```
Comment (M03ED) [inline app/utils/report.py:155]: i can't find this function in discord module

**Review by M03ED — CHANGES_REQUESTED** [review]

**Comment by ImMohammad20000** [comment]
بنظرم خوبه مرج کنیم

**Review by M03ED — APPROVED** [review]

**Pull request merged**